src/Orleans.Streaming/PersistentStreams/PersistentStreamPullingAgent.cs

```diff
@@ -12,11 +12,12 @@
 using Orleans.Timers;
 
 namespace Orleans.Streams
-{
+{ 
     internal class PersistentStreamPullingAgent : SystemTarget, IPersistentStreamPullingAgent
     {
-        private static readonly IBackoffProvider DeliveryBackoffProvider = new ExponentialBackoff(TimeSpan.FromSeconds(1), TimeSpan.FromSeconds(30), TimeSpan.FromSeconds(1));
-        private static readonly IBackoffProvider ReadLoopBackoff = new ExponentialBackoff(TimeSpan.FromSeconds(10), TimeSpan.FromSeconds(20), TimeSpan.FromSeconds(1));
+        private readonly IBackoffProvider _deliveryBackoffProvider;
```

Comment:
I do prefer using `_` for private member, but for consistency we should keep the same convention already used in this file